docs/platforms/javascript/common/configuration/integrations/featureFlags.mdx

```diff
@@ -0,0 +1,70 @@
+---
+title: Feature Flags
+description: "Learn how to attach custom feature flag data to Sentry error events."
+notSupported:
+  - javascript.aws-lambda
+  - javascript.azure-functions
+  - javascript.bun
+  - javascript.capacitor
+  - javascript.cloudflare
+  - javascript.connect
+  - javascript.cordova
+  - javascript.deno
+  - javascript.electron
+  - javascript.express
+  - javascript.fastify
+  - javascript.gcp-functions
+  - javascript.hapi
+  - javascript.koa
+  - javascript.nestjs
+  - javascript.nodejs
+  - javascript.wasm
+---
+
+<PlatformContent includePath="feature-flags/prerelease-alert" />
+
+<Alert level="info">
+
+This integration only works inside a browser environment.
+
+</Alert>
+
+The Feature Flags integration allows you to manually track feature flag evaluations through an API. These evaluations are held in memory, and in the event an error occurs, sent to Sentry for review and analysis.
+**At the moment, we only support boolean flag evaluations.**
+
+_Import names: `Sentry.featureFlagsIntegration` and `type Sentry.FeatureFlagsIntegration`_
+
+## Install
+
+Install your platform's Sentry SDK from npm.
+
+## Configure
+
+```JavaScript
+import * as Sentry from '@sentry/<your browser platform, e.g. react>';
```

Comment:
😬 you'll want to have install code for each platform unfortunately and then plop something like this in here:

```
<PlatformContent includePath="feature-flags/install" />
```

Reply:
oof we need this for other FF integrations too. Maybe there's an existing platform content for it 👀
Or maybe we can exclude this line since we already have the "Import names"?